blocks/api/post.pegjs

```diff
@@ -5,6 +5,13 @@
 // are the same as `json_decode`
 ?> **/
 
+function freeform( s ) {
+  return s.length && {
+    blockName: 'core/freeform',
+    innerHtml: s
```

Comment:
We might want to consider the capitalization here. I created #2511 a while back to open discussion on a convention. I've started leaning more toward capitalizing abbreviations (`innerHTML`) but really just care to enforce some consistency.

Reply:
personally I don't care. I'll use `HTML`

Reply:
changed in 4e41437d